classes.py

```diff
@@ -0,0 +1,7 @@
+class Process:
```

Comment:
I'd be wary of naming this class `Process` since it's likely a name that's already being used by Python. I recommend naming it SolarCarProcess